apps/server/src/shared/domain/domainobject/board/file-element.do.ts

```diff
@@ -0,0 +1,25 @@
+import { BoardComposite, BoardCompositeProps } from './board-composite.do';
+import type { AnyBoardDo } from './types';
+import type { BoardNodeBuildable } from './types/board-node-buildable';
+import type { BoardNodeBuilder } from './types/board-node-builder';
+
+export class FileElement extends BoardComposite implements FileElementProps, BoardNodeBuildable {
```

Comment:
Does the word Buildable as a none exist? I think it's only an adjective and not a none. Wouldn't it be `BoardNodeBuild`?